docs/network-spec/connection-manager.tex

```diff
@@ -1705,53 +1713,144 @@ \section{Inbound Protocol Governor}
+                                                           {\MiniProtocolTerminated}  (RemoteWarm);
+      \draw[->] (RemoteHot)        edge[loop right]    node[right]
+                                                           {\MiniProtocolTerminated}  (RemoteHot);
+
+      \draw[->] (RemoteHot.70)     to [out=60,in=300]   node[fill=white,right=-25pt,pos=0.4]
+                                                            {\DemotedToWarmRemote}    (RemoteWarm.290);
+      \draw[->] (RemoteWarm.250)   to [out=240,in=120]  node[fill=white,left=-80pt,pos=0.4]
+                                                            {\PromotedToHotRemote}    (RemoteHot.110);
+    \end{tikzpicture}
+    \caption{Inbound protocol governor state machine}
+    \label{fig:inbgov-state-machine}
+  \end{center}
 \end{figure}
 
-There is a map \(\Phi\) from the sub-graph of connection manager state machine
-which represents negotiated inbound or duplex outbound connections to the
-\inbgov{} state machine. \(\Phi\) is specified by the following table:
-\begin{align*}
-  & \Phi(\text{\InboundIdleStateAny}) & =\; & \text{\RemoteIdle}\\
-  & \Phi(\text{\OutboundStateDupTau}) & =\; & \text{\RemoteIdle}\\
-  & \Phi(\text{\OutboundStateDup})    & =\; & \text{\RemoteCold} \\
-  & \Phi(\text{\DuplexState})         & =\; & \text{\RemoteEstablished} \\
-  & \Phi(\text{\InboundStateAny})     & =\; & \text{\RemoteEstablished} \\
-  & \Phi(\text{\TerminatingState})    & =\; & \text{\FinalState} \\
-  & \Phi(\text{\TerminatedState})     & =\; & \text{\FinalState} \\
-\end{align*}
-Furthermore, \(\Phi\) gives rise to a functor: any transformation that has
-\textsf{Local} subscript is mapped to identity:
-\begin{lstlisting}
-*'$\Phi$(\textsf{a\textsubscript{Local}}) = \textsf{id}'*
-*'$\Phi$(\DemotedToColdAnyRem:\InboundIdleStateAny{}$\rightarrow$\TerminatingState)'*
-  = *'$\text{\CommitRemote}\circ\text{\TimeoutExpired}$'*
-*'$\Phi$(\DemotedToColdAnyRem:\DuplexState{}$\rightarrow$\OutboundStateDupTau)'*
-  = *'\RemoteToCold'*
-*'$\Phi$(\PromotedToWarmAnyRem) = \AwakeRemote'*
-*'$\Phi$(\TimeoutExpired)       = \TimeoutExpired'*
-*'$\Phi$(\AwakeAnyRem)          = \AwakeRemote'*
-\end{lstlisting}
+\subsection{States}
+
+The state of the inbound governor are similar to the outbound governor, but
+there are crucial differences.
+
+\subsubsection{\RemoteCold}
+The remote cold state segnifies that the remote peer is not using the
+connection, however the only reason why the inbound governor needs to track
+that connection is because the outbound side of this connection is used.  The
+inbound governor will wait until any of the responder mini-protocols wakes up
+(\AwakeRemote{}) or the mux will be shutdown (\MuxTerminated{}).
+
+\subsubsection{\RemoteIdle}
+The \RemoteIdle{} state is an intial state of each connection
+(\NewConnection{}).  An active connection will become \RemoteIdle{} once the
+inbound governor will detect that all responder mini-protocols terminated
```

Comment:
once the inbound governor will detect that all responder mini-protocols terminated -> once the inbound governor detects that all responder mini-protocols terminated